apps/native/app/src/graphql/queries/profile.graphql

```diff
@@ -13,6 +13,12 @@ query GetProfile {
   }
 }
 
+query GetUserLocale {
+  getUserProfile {
```

Comment:
I wonder if we are already making the round trip to user profile if we should fetch it here and then load from the cache in settings

Reply:
yes could be a good idea, will update :)